color/src/floatfuncs.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 the Color Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Shims for math functions that ordinarily come from std.
+
+/// Defines a trait that chooses between libstd or libm implementations of float methods.
+macro_rules! define_float_funcs {
+    ($(
+        fn $name:ident(self $(,$arg:ident: $arg_ty:ty)*) -> $ret:ty
+        => $lname:ident/$lfname:ident;
```

Comment:
`lname` doesn't seem to be used?

Reply:
Yup, these were adapted from kurbo, but I think we can simplify and only include f32 versions.